mscore/default.txt

```diff
@@ -0,0 +1,11 @@
+Clef
+    Treble Clef
+    Base Clef
+    Alto Clef
+    Tenor Clef
+
+Key Signature
+    G Minor
+    B Major
+    F Minor
+    C Major
```

Comment:
We don't need this file in MuseScore: use the data from the xmlReader instead. This file is useful for your example project so you can experiment with models and views without worrying about MuseScore, but that's all.

Reply:
OK, I see that you just added it as a way to check that the TreeView was working. [This comment](https://github.com/musescore/MuseScore/pull/5135#r294494602) explains how to get this file working, but that is not what was stopping the view from appearing. Even without any data, the TreeView should still appear (it will just be empty). These comments explain what you need to do to the [model](https://github.com/musescore/MuseScore/pull/5135#r294481985) and the [view](https://github.com/musescore/MuseScore/pull/5135#r294489246) to make it all work as expected.